README.md

```diff
@@ -274,6 +276,15 @@ PROMPT KEYS:
     q, ^D, double Enter    exit buku
 ```
 
+#### Webserver options
+
+Your bookmark on buku can be accesed through browser. To run the server on host 0.0.0.1  on port 5001, run following command:
```

Comment:
Please link to `bukuserver/README.md` and add the details there. I think we can take all the details from tom's PR.